### PR TITLE
Flyway integration

### PR DESCRIPTION
**Pull request by rabadiyaronak**

### CODE REVIEW

File: src/main/java/com/epam/jpop/userms/model/ErrorResponse.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
```
Comment (samalprasant123) [inline src/main/java/com/epam/jpop/userms/model/ErrorResponse.java]: Use `@Data`

File: src/main/java/com/epam/jpop/userms/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        return userService.getAllUsers();
+    }
+
+    @GetMapping("/{id}")
+    public UserDetail getById(@PathVariable Long id) {
+        return userService.getUserById(id);
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> add(@RequestBody UserDetail userDetail) {
```
Comment (samalprasant123) [inline src/main/java/com/epam/jpop/userms/controller/UserController.java]: Use @Valid

File: src/test/java/com/epam/jpop/userms/service/impl/UserServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    @Mock
+    private UserRepository userRepository;
+
+    @InjectMocks
+    private UserServiceImpl userService;
+
+    @Test
+    void getAllUsers() {
+        List<User> users = prepareUserMockData();
```
Comment (samalprasant123) [inline src/test/java/com/epam/jpop/userms/service/impl/UserServiceImplTest.java:31]: //given
`<code here>`
//when
`<code here>`
//then
`<code here>`

Follow this pattern

**Review by samalprasant123 — APPROVED** [review]

File: src/main/java/com/epam/jpop/userms/model/ErrorResponse.java (hunk 1 above)
Comment (rabadiyaronak) [reply src/main/java/com/epam/jpop/userms/model/ErrorResponse.java]: Completed

File: src/test/java/com/epam/jpop/userms/service/impl/UserServiceImplTest.java (hunk 3 above)
Comment (rabadiyaronak) [reply src/test/java/com/epam/jpop/userms/service/impl/UserServiceImplTest.java:31]: resolved

**Pull request merged**